test/extended/util/framework.go

```diff
@@ -105,6 +196,36 @@ func WaitForOpenShiftNamespaceImageStreams(oc *CLI) error {
 					return false
 				}
 			}
+			// check the samples operator to see about imagestream import status
```

Comment:
doing this check inside the "for each lang" loop doesn't seem to make sense?  you just need to do it once, right?

Reply:
yep part of next push